### PR TITLE
Fix component imports within components package

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Instead of referencing `@tektoncd/dashboard-components`, imports
within the `components` package that reference other components
should load them locally from the index. This will be more important
in future when we provide the ability to override / customise parts
of the UI via extensions.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/main/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@tektoncd/dashboard-utils": "file:../utils",
+    "classnames": "^2.3.1",
```
Comment (briangleeson) [inline packages/components/package.json:22]: This is only used within components and graphs packages. So we could remove it from the main package.json? https://github.com/tektoncd/dashboard/blob/main/package.json#L31

Comment (AlanGreene) [reply packages/components/package.json:22]: Absolutely, I thought I had done that with this change but clearly not. Good catch!

Comment (AlanGreene) [reply packages/components/package.json:22]: Updated

**Review by briangleeson — APPROVED** [review]
/lgtm

**Pull request merged**